padglobal/view/which.py

```diff
@@ -0,0 +1,81 @@
+from discordmenu.embed.base import Box
+from discordmenu.embed.components import EmbedMain, EmbedField
+from discordmenu.embed.text import Text
+from discordmenu.embed.view import EmbedView
+from discordmenu.emoji.emoji_cache import emoji_cache
+from tsutils import embed_footer_with_state
+
+STAR = '* '
+BULLET_EMOJIS = ['db', 'dp', 'dg']
+
+
+class WhichViewProps:
+    def __init__(self, name: str, definition: str, timestamp: str, success: bool):
+        self.name = name
+        self.definition = definition
+        self.timestamp = timestamp
+        self.success = success
+
+
+def _get_field(paragraph, color):
+    lines = paragraph.strip().splitlines()
+    lines = [_replace_bullets(line, color) for line in lines]
+
+    if len(lines) > 1:
+        evo = lines[0]
+        information = '\n'.join(lines[1:])
+        if evo.startswith('**'):
+            return EmbedField(evo, Box(information))
+
+    # it wasn't multiple lines that started with a bolded line, assume it's not an evo description
+    return EmbedField('**Additional Information**', Box(paragraph))
+
+
+def _replace_bullets(line, color_index):
+    if line.startswith(STAR):
+        bullet = emoji_cache.get_emoji(BULLET_EMOJIS[color_index])
+        return '{} {}'.format(bullet, line[len(STAR):])
+    else:
+        return line
+
+
+def get_description(definition, timestamp, success):
+    if success:
+        return Text('Last Updated {}'.format(timestamp))
```

Comment:
after all that work to change the timestamps can we just say like "No last-updated date recorded" if the date is set to the 1969 date, this is probably the clearest option